py/pykms/pykmsutil.cpp

```diff
@@ -26,10 +26,13 @@ void init_pykmstest(py::module &m)
 			.def("reserve_connector", (Connector* (ResourceManager::*)(const string& name))&ResourceManager::reserve_connector,
 			     py::arg("name") = string())
 			.def("reserve_crtc", &ResourceManager::reserve_crtc)
-			.def("reserve_plane", &ResourceManager::reserve_plane,
+			.def("reserve_plane", (Plane* (ResourceManager::*)(Crtc*, Plane::Type, PixelFormat)) &ResourceManager::reserve_plane,
 			     py::arg("crtc"),
```

Comment:
You are right, but I pushed a new version already.
I'll make yet another push and squash those two together, but I first wait if you have any more comments to this latest version.